R/automation.R

```diff
@@ -102,6 +102,10 @@ setMethod("scriptSavepoint", "Script", function(x) {
 #' @param is_file The default guesses whether a file or string was
 #' used in the `script` argument, but you can override the heuristics
 #' by specifying `TRUE` for a file, and `FALSE` for a string.
+#' @param encoding The encoding the script file is saved as on your
+#' machine. Defaults to UTF-8.
```

Comment:
```suggestion
#' @param encoding Optional encoding to convert \strong{from}: the API accepts only UTF-8.
```

Reply:
I did these locally so that I could re-roxygen. Thanks!